app/uk/gov/hmrc/economiccrimelevyregistration/views/AmendRegistrationPdfView.scala.html

```diff
@@ -38,7 +38,7 @@ <h2>@messages("checkYourAnswers.organisationDetails")</h2>
     @govukSummaryList(viewModel.organisationDetails())
 
 
-    @if(viewModel.hasSecondContact) {
+    @if(viewModel.hasSecondContact && !viewModel.hasSecondContactDetailsPresentChanged) {
```

Comment:
Make this it's own appropriately named viewModel property/val

Reply:
hasSecondContactDetailsPresentChanged is already it's own val in TrackRegistrationChanges and is unit tested